sources/platform/actors/publishing/monetize.mdx

```diff
@@ -71,22 +74,82 @@ You make your Actor pay-per-result and set the price to be $1/1,000 results. Dur
 Your profit is computed only from the first two users, since they are on Apify paid plans. The revenue for the first user is $50 and for the second $20, i.e., total revenue is $70. The total underlying cost is _$5 + $2 = $7_. Since your profit is 80% of the revenue minus the cost, it would be _0.8 * 70 - 7 = $49_.
 </details>
 
-#### Best practices for Pay-per-results Actors
+#### Best practices for PPR Actors
 
 To ensure profitable operation:
 
 - Set memory limits in your [`actor.json`](https://docs.apify.com/platform/actors/development/actor-definition/actor-json) file to control platform usage costs
 - Implement the `ACTOR_MAX_PAID_DATASET_ITEMS` check to prevent excess result generation
 - Test your Actor with various result volumes to determine optimal pricing
 
+### Pay-per-event pricing model
+
+The pay-per-event pricing model offers a flexible monetization option for Actors on Apify Store. Unlike the pay-per-result, PPE allows you to charge users based on specific events triggered programmatically by your Actor's code.
+
+#### PPE vs PPR
+
+Unlike PPR, which charges based on the number of results produced, PPE lets you define pricing for individual events. You can charge for specific events directly from your Actor by calling the PPE charging API. Common events include Actor start, dataset item creation, and external API calls.
+
+For a detailed comparison of pricing models, refer to our [user-facing documentation](/platform/actors/running/actors-in-store).
+
+#### How is profit computed
+
+Your profit is calculated as follows:
+
+`profit = (0.8 * revenue) - platform costs`
+
+where:
+
+- _Revenue_: The amount charged for events via the PPE [API](/api/v2/post-charge-run) or through [JS](/sdk/js/reference/class/Actor#charge)/[Python](/sdk/python/reference/class/Actor#charge) SDK. You receive 80% of this revenue.
+- _Platform costs_: The underlying platform usage costs for running the Actor, calculated using the same unit pricing as PPR.
+
+Only paid user activity is included in profit calculations.
+
+#### How to set pricing for PPE
+
+1. _Understand your costs_: Analyze resource usage (e.g CPU, memory, proxies, external APIs) and identify cost drivers
+1. _Define clear events_: break your Actor's functionality into measurable, chargeable events.
+1. _Common use cases_:
+   1. _For scraping_: combine Actor start and dataset items pricing to reflect setup and per-result cost.
+   1. _Beyond scraping_: Account for intergrations with external systems or external API calls.
+1. _External API costs_: Account for additional processing costs.
+1. _Test your pricing_: Run your Actor and analyze cost-effectiveness using a special dataset.
+1. _Communicate value_: Ensure pricing reflects the value provided and is competetive.
+
+#### Best practices for PPE Actors
```

Comment:
I added some practices to include in my last review, can you please check the list? Thx